events.md

```diff
@@ -6,23 +6,23 @@
 #### Upcoming Events
 
 January 14, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
 - **Join Event:** https://primetime.bluejeans.com/a2m/live-event/dersrzsx
 
 February 11, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
 - **Join Event:** https://primetime.bluejeans.com/a2m/live-event/dersrzsx
 
 March 11, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
```

Comment:
Daylight saving might be an issue in March. Better to check the exact time.

Reply:
good point, so US summer change in March 14 and Europe change at March 28, so March 25th would be the only one that special

Reply:
good point, verified by https://www.timeanddate.com/worldclock/meetingtime.html?month=5&day=6&year=2021&p1=283&p2=268&iv=0, after March 25th would be 09:00 - 10:00 PT / 16:00 - 17:00 GMT